packages/Panel/stories/Panel.examples.stories.js

```diff
@@ -38,6 +39,14 @@ export const basic = () => (
 
 basic.story = sidePanelStoryParameters;
 
+export const coplanarPanel = () => (
+  <ExampleStory component="Panel" storyName="Co-Planar Panel" fileName="examples/CoPlanarPanel.js">
+    <CoPlanarPanel />
+  </ExampleStory>
+);
+
+coplanarPanel.story = sidePanelStoryParameters;
```

Comment:
I recommend this instead:

```js
showcase.story = {
  name: "Co-Planar Panel",
  parameters: exampleStoryParameters,
};
```

In fact that pater could be used for all the stories in this file and then we don't even need `sidePanelStoryParameters`.